packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -33,9 +33,41 @@
   overflow: hidden;
 }
 
+.mdc-chip--with-leading-icon {
+  padding-top: $mdc-chip-with-icon-padding;
+  padding-bottom: $mdc-chip-with-icon-padding;
+  padding-left: $mdc-chip-with-icon-padding;
+}
+
+.mdc-chip--with-trailing-icon {
+  padding-right: $mdc-chip-with-icon-padding;
+}
+
 .mdc-chip__text {
   display: inline-block;
   vertical-align: middle;
   // Set line-height to be <18px to force the content height of mdc-chip to be 18px.
   line-height: 17px;
 }
+
+.mdc-chip__icon {
+  color: $mdc-chip-icon-color;
+  vertical-align: middle;
+  margin: 0 4px;
+  border-radius: 16px;
+  outline: none;
+}
+
+.mdc-chip__icon.mdc-chip__icon--trailing {
```

Comment:
I don't think most of these rules need the double class selector here?

We might want to use 2 classes for font-size specifically though, given the snag Brendan ran into when he can't guarantee ordering of material-icons and MDC stylesheets.

Reply:
Based on a conversation I just had with Brendan, keep the width/height/font-size with the 2-class selector... the hover and focus are probably safe to just have `.mdc-chip__icon--trailing` though.

Reply:
Why do we need both classes? All of the chips CSS rules are in a single file, so CSS load order shouldn't be a problem. What am I missing?

Reply:
material-icons is not in the same file, or even the same repository. See #2242.

Reply:
Gotcha. Can we add a comment above each 2-class-selector explaining why we need the extra class?